src/content/docs/highlights/v0.30.md

```diff
@@ -0,0 +1,692 @@
+---
+title: v0.30.0
+---
+
+![animation](../../../assets/ratatui-animation.gif)
+
+We are excited to announce Ratatui 0.30.0!
+
+See the [changelog](https://github.com/ratatui/ratatui/blob/main/CHANGELOG.md) for the full list of
+changes.
```

Comment:
Here would be good to add a paragraph pointing at the major items in the highlights "We [Modularized](#Modularization) the crates. Blah [now supports blah](#foo)"